docs/sdk/migrations/3.2.0.md

```diff
@@ -0,0 +1,126 @@
+---
+sidebar_position: 2
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Migrating from 3.1.x to 3.2.0
+
+# Removed `controllerSessionSigs`
+
+Instead of receiving the `sessionSigs` directly, now they will obtain it from the client and its auth context. This way we are unifying that management and using them from a common storage instead of making the user handle it directly. For more information, please check this [PR](https://github.com/LIT-Protocol/js-sdk/pull/358) 
```

Comment:
Maybe clarify that the affected entities are the ones starting with `PKP...` including `PKPEthersWallet` or `PKPClient`